Makefile

```
@@ -185,23 +188,48 @@ generate: controller-gen
 ### docker: Builds and pushes controller image
 docker: _print_vars docker-build docker-push
 
-### docker-build: Builds the controller image
-docker-build:
-	$(DOCKER) build . -t ${DWO_IMG} -f build/Dockerfile
+### docker-build: Builds the images and the local manifest list
+docker-build: _docker-build-amd64 _docker-build-arm64
+	# clean up any old local manifest list to avoid conflicts
+	$(DOCKER) manifest rm ${DWO_IMG} || true
+	# create a new local manifest list from the freshly built images
+	$(DOCKER) manifest create ${DWO_IMG} ${DWO_IMG}-amd64 ${DWO_IMG}-arm64
+
+
+### _docker-build-amd64: Builds the amd64 image
+_docker-build-amd64:
+	$(DOCKER) build . --platform linux/amd64 -t ${DWO_IMG}-amd64 -f build/Dockerfile
+
+### _docker-build-arm64: Builds the arm64 image
+_docker-build-arm64:
+	$(DOCKER) build . --platform linux/arm64 -t ${DWO_IMG}-arm64 -f build/Dockerfile
```

@akurinnoy should it be

```suggestion
	$(DOCKER) build . --platform linux/arm64 -t ${DWO_IMG}-arm64 -f build/Dockerfile.arm64
```

?